multipatch_analysis/pipeline/multipatch/dataset.py

```diff
@@ -195,6 +196,7 @@ def create_db_entries(cls, job, session):
                             data_start_time=resampled.t0,
                             ex_qc_pass=resp['ex_qc_pass'],
                             in_qc_pass=resp['in_qc_pass'],
+                            meta=resp['qc_failures'],
```

Comment:
To allow meta to expand in the future, this should be `{'qc_failures': resp['qc_failures']}`